lambda/query-activity-log/models.ts

```diff
@@ -0,0 +1,43 @@
+type UrnFdnSub = string;
```

Comment:
why are some fields pulled out and others not?

Reply:
That's a good point not sure why it is needed . Happy for it to be all in the same format .